### PR TITLE
commenting on Issues/PRs with ogr

### PR DESCRIPTION
**Pull request by marusinm**
Commenting on Issues and PRs are available for Github but also Pagure since now. I added two new variables into the private config file `pagure_token` and `pagure_username`. I'm just thinking if we should add this into README or wait till all Github features will be implemented also for Pagure.

Ogr doesn't have released commenting on issues yet. If you want to try locally you must install ogr from Github, not from requirements.txt
`pip install git+https://github.com/packit-service/ogr@master`

### CODE REVIEW

File: release_bot/configuration.py [hunk 1]
```diff
@@ -179,5 +185,54 @@ def get_pypi_project_from_setup_cfg(setup_cfg):
...
+        :return: ogr Github/Pagure project instance or None
+        """
+        if self.github_token != '' and self.github_username != '':
+            return self.get_github_project(username=self.github_username,
+                                           repository=self.repository_name,
+                                           token=self.github_token)
+        elif self.pagure_token != '' and self.pagure_username != '':
+            return self.get_pagure_project(username=self.pagure_username,
+                                           repository=self.repository_name,
+                                           token=self.pagure_token)
```
Comment (lachmanfrantisek) [inline release_bot/configuration.py]: [This PR](https://github.com/packit-service/ogr/pull/95) should help you with that..;-) Feel free to add some comments. I would appreciate some opinions from the perspective of release-bot.

We can release a new version of OGR after merging that...

**Comment by marusinm** [comment]
@jpopelka This definitely doesn't fully implement #190, I will go through release-bot functions and make smaller PRs to complete #190. This also needs to contribute to ogr.

I also deleted the test for adding a comment on Issue/PR since it is already tested in ogr. However, we will need to provided integration tests for Pagure and Github. But firstly, I assume to replace more API calls with ogr.

File: release_bot/configuration.py (hunk 1 above)
Comment (marusinm) [reply release_bot/configuration.py]: My current solution is easier for required options `github_token` and `github_username` in `conf.yaml`.  Your [PR](https://github.com/packit-service/ogr/pull/95) creates ogr project instance from url. I can use it, but first, we need to discuss #230. Option `clone_url` in `conf.yaml` must be used. 

Sorry, I missed adding a comment to your [PR](https://github.com/packit-service/ogr/pull/95).

Comment (lachmanfrantisek) [reply release_bot/configuration.py]: We can discuss that in #230 ...

File: release_bot/configuration.py [hunk 2]
```diff
@@ -20,6 +20,9 @@
 from pathlib import Path
 
 import yaml
+from ogr.services.github import GithubService
+from ogr.services.pagure import PagureService
+from ogr.factory import get_project
```
Comment (lachmanfrantisek) [inline release_bot/configuration.py]:
```suggestion
from ogr import get_project
```

File: release_bot/configuration.py [hunk 3]
```diff
@@ -20,6 +20,9 @@
 from pathlib import Path
 
 import yaml
+from ogr.services.github import GithubService
+from ogr.services.pagure import PagureService
```
Comment (lachmanfrantisek) [inline release_bot/configuration.py]:
```suggestion
from ogr import PagureService
```

File: release_bot/configuration.py [hunk 4]
```diff
@@ -20,6 +20,9 @@
 from pathlib import Path
 
 import yaml
+from ogr.services.github import GithubService
```
Comment (lachmanfrantisek) [inline release_bot/configuration.py]: This should work as well:

```suggestion
from ogr import GithubService
```

File: release_bot/configuration.py [hunk 5]
```diff
@@ -179,5 +188,17 @@ def get_pypi_project_from_setup_cfg(setup_cfg):
...
+        """
+        Create ogr project instance based on provided configuration.
+        Project instance is used for manipulating with Github/Pagure repo.
+        :return: ogr Github/Pagure project instance or None
+        """
+        return get_project(url=self.clone_url,
+                           custom_instances=[
+                               GithubService(token=self.github_token),
+                               PagureService(token=self.pagure_token,
+                                             instance_url=self.pagure_instance_url)])
```
Comment (lachmanfrantisek) [inline release_bot/configuration.py]: :+1:

**Pull request merged**